noir-projects/aztec-nr/aztec/src/keys/getters.nr

```diff
@@ -1,79 +1,98 @@
-use dep::protocol_types::{
-  address::{
-    AztecAddress,
-    PartialAddress
-  }, 
-  constants::{
-    GENERATOR_INDEX__PUBLIC_KEYS_HASH,
-    GENERATOR_INDEX__CONTRACT_ADDRESS_V1,
-    CANONICAL_KEY_REGISTRY_ADDRESS
-  },
-  grumpkin_point::GrumpkinPoint,
+use dep::protocol_types::{address::AztecAddress, constants::CANONICAL_KEY_REGISTRY_ADDRESS, grumpkin_point::GrumpkinPoint};
+use crate::{
+    context::PrivateContext, hash::poseidon2_hash, oracle::keys::get_public_keys_and_partial_address,
+    state_vars::{
+    map::derive_storage_slot_in_map,
+    shared_mutable::shared_mutable_private_getter::SharedMutablePrivateGetter
+}
 };
 
-use crate::context::PrivateContext;
-use crate::hash::{
-  pedersen_hash, 
-  poseidon2_hash,
-};
-use crate::oracle::keys::get_public_keys_and_partial_address;
-use crate::state_vars::{
-  map::derive_storage_slot_in_map,
-  shared_mutable::shared_mutable_private_getter::SharedMutablePrivateGetter,
-};
+// Note: In fetch_key_from_registry we expect that shared mutable slow is index * 2 + 1 for x coordinate
+// and index * 2 + 2 for y coordinate, if this changes the function will need to be refactored.
```

Comment:
It's not clear to me which indices this refers to :thinking:

Reply:
Addressed with Esau's help in https://github.com/AztecProtocol/aztec-packages/pull/6171/commits/d338aad76074ff2f5c54a1520bf5c57b49e8fe86